packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -290,7 +417,59 @@ const Root = ({
       >
         {intl.formatMessage(formMessages.repeaterSkipTo)}
       </Link>
-      {children}
+      {children && <div data-h2-margin-bottom="base(x.5)">{children}</div>}
+      {approachingLimit && (
+        <Well
+          data-h2-margin-bottom="base(x1)"
+          data-h2-text-align="base(center)"
+        >
+          <p data-h2-font-weight="base(700)" data-h2-margin-bottom="base(x.5)">
+            {intl.formatMessage(formMessages.approachingLimit)}
+          </p>
+          <p>{intl.formatMessage(formMessages.approachingLimitDetails)}</p>
+        </Well>
+      )}
+      {total === 0 && (
+        <Well
+          data-h2-margin-bottom="base(x1)"
+          data-h2-text-align="base(center)"
+        >
+          {customNullMessage ?? (
+            <>
+              <p
+                data-h2-font-weight="base(700)"
+                data-h2-margin-bottom="base(x.5)"
+              >
+                {intl.formatMessage(formMessages.repeaterNull)}
+              </p>
+              <p>{intl.formatMessage(formMessages.repeaterNullDetails)}</p>
+            </>
+          )}
+        </Well>
+      )}
+      {hasError && (
+        <Field.Error id={name} data-h2-padding="base(0)">
+          <Well
+            data-h2-text-align="base(center)"
+            data-h2-margin-bottom="base(x1)"
+          >
```

Comment:
Should be safe to drop this well and move the `text-align` to the error element dropping the `padding: 0;` at the same time. This should avoid the overlapping blue/red borders.

![Screenshot 2023-10-13 120434](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/54c80dc7-de27-4624-b8ad-49bb3f3c0037)

Reply:
Nice catch! That's new, I must have missed it with last changes. Fixed by [e51e6ab](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/e51e6ab11f1085af2d7a5fa68c9f549426c83db3).

Reply:
Looks great! Sorry, I also noticed we don't have a period, should we add one?

Reply:
Fixed by [6f56faa](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/6f56faa8542558ac5c339179a8b4fe76b708c53a)